### DISCUSSION TITLE
z/OS Open Source Guild Meeting #8 - March 29, 2023 at 10am EST

### DISCUSSION THREAD

**Discussion by IgorTodorovskiIBM** [zopencommunity/meta#237] [Guild]
### Agenda

* Highlights in z/OS Open Source since last month (10 mins)
  * [Progress report](https://github.com/ZOSOpenTools/meta/blob/main/docs/Progress.md)
  * Introduce new tools/libraries (lynx)
  * Improvements to existing tooling
* Introducing the latest in zopen (Russell Wilson - @DevonianTeuchter)
  * Installing tools to a common directory
  * zopen query
  * zopen log
  * and more!
* Upcoming plans to migrate to Clang
* Improving Security discussion
* Survey and Open Discussion (10 mins)

### How to Join?
Join link: https://ibm.webex.com/ibm/j.php?MTID=m72857a0933fbcd1c58c116a13c9f1e7b 
Webinar number: 145 105 4264
Webinar password: QamxS8QX2m7 (72697879 from phones)

### Date and Time
* March 29, 10am-11pm EST (7am PST)
* **New:** We will also be holding a session for the Asia-Pacific audience. Details on the APAC meeting to follow.

### Recording/Slide/Poll/Chat Log
* [Open Source Guild Mar 2023_final.pdf](https://github.com/ZOSOpenTools/meta/files/11104133/Open.Source.Guild.Mar.2023_final.pdf), 
[Open Source Guild Mar 2023_final.pptx](https://github.com/ZOSOpenTools/meta/files/11104125/Open.Source.Guild.Mar.2023_final.pptx)
* [Poll Results](https://github.com/ZOSOpenTools/meta/files/11104129/poll-results-8.pdf)
* [Recording](https://ibm.ent.box.com/s/4s7mcf804opx5s9fm6qs2io9tgxveeor)
* [Chag log](https://github.com/ZOSOpenTools/meta/files/11104217/chat_guild8.csv)

**Comment by lbdyck** [comment]
Any chance someone can create an ics file for the guild meetings so we can just click to add to our calendar?

**Reply by IgorTodorovskiIBM** [reply]
> Any chance someone can create an ics file for the guild meetings so we can just click to add to our calendar?

Good idea, does this work for you? (unfortunately github does not like .ics format so I had to zip it).
[invite.ics.zip](https://github.com/ZOSOpenTools/meta/files/11090225/invite.ics.zip)

**Reply by lbdyck** [reply]
Perfection......

**Comment by IgorTodorovskiIBM** [comment]
* Recording, slides, and poll results posted in the description
